scheduler/src/cook/kubernetes/api.clj

```diff
@@ -643,6 +643,24 @@
                                      (.withEmptyDir (V1EmptyDirVolumeSource.)))]
     (.build builder)))
 
+(defn make-shm-volume
+  "Make a volume for /dev/shm and add it to the volume mount for the given image"
+  []
+  (let [^V1EmptyDirVolumeSource volume-source (V1EmptyDirVolumeSource.)
+        _ (.medium volume-source "Memory")
+        ^String name (str "shmvolume-" (+ (rand-int 899999) 100000))
```

Comment:
Why do we need the randomness here?

Reply:
The idea is to avoid accidental name collision if someone else makes a volume with a similar name.

Reply:
Ah, got it. Maybe a comment explaining this would be helpful?

Reply:
Done.